client/me/help/style.scss

```diff
@@ -1,3 +1,7 @@
+.help .upwork-banner {
+	margin-bottom: 24px;
```

Comment:
Can we please change this to `margin: 10px 10px 20px;` instead so the `x` doesn't sit too close next to the Notification bell icon at `<660px`? It's a really small tap target.

Instead of:

![screen shot 2019-03-06 at 9 59 01 am](https://user-images.githubusercontent.com/4924246/53902740-b9030b80-3ff6-11e9-9c90-2d669ad03fb1.png)

It'll look like:

![screen shot 2019-03-06 at 10 01 21 am](https://user-images.githubusercontent.com/4924246/53902784-cfa96280-3ff6-11e9-8e7f-7f9f0ea4afc2.png)

Thanks @belcherj.

Reply:
done